### DISCUSSION TITLE
Which contract has Optimism's proposals?

### DISCUSSION THREAD

**Discussion by Eithcowich** [ethereum-optimism/developers#182] [Application Development] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I'm from DeepDAO. We want to integrate the new Optimism proposals (read only for now) and not sure where to find them. We already looked at the [proxy contract](https://optimistic.etherscan.io/address/0xcdf27f107725988f2261ce2256bdfcde8b382b10#readProxyContract) mentioned in one of the documents, but not sure which events represent a proposal creation. 

You can see the proposals we already integrated [here](https://deepdao.io/organization/10edb97a-9dba-448a-88b1-ca263a7c75ad/organization_data/governance). Comparing it to Agora, we're missing the last 2 proposals. 

Thanks.

**Comment by sbvegan** [comment]
I've asked about this internally, I'll see what I can find out.

**Comment by Eithcowich** [comment]
Hey @sbvegan any news on this?

**Reply by sbvegan** [reply]
Yeah sorry I was off on a goose chase for this one. It looks like the proposals go through [Agora](https://vote.optimism.io/) and the proxy contract that manages the proposals is [0xcDF27F107725988f2261Ce2256bDfCdE8B382B10](https://optimistic.etherscan.io/address/0xcDF27F107725988f2261Ce2256bDfCdE8B382B10#readProxyContract).

**Reply by sbvegan** [reply]
What do you mean by, they're missing the last two proposals?

**Comment by Eithcowich** [comment]
> `Yeah sorry I was off on a goose chase for this one. It looks like the proposals go through [Agora](https://vote.optimism.io/) and the proxy contract that manages the proposals is [0xcDF27F107725988f2261Ce2256bDfCdE8B382B10](https://optimistic.etherscan.io/address/0xcDF27F107725988f2261Ce2256bDfCdE8B382B10#readProxyContract).`

Hey @sbvegan thanks for chasing for it. This contract gives some of the proposal details, but not the main attributes we need: 

`proposal id`
`title`
`description`
`transactions to be executed`

Where can we find these? 

Thanks!

**Reply by yitongzhang** [reply, marked as answer]
Hey @Eithcowich thanks for reaching out! we love DeepDAO and would love to support – I'm from Agora, and we maintain the Optimism Governor and voting application. You can find the full documentation for governor in this repo. The attributes you're looking for should be documented here: 

https://github.com/voteagora/optimism-gov

If you send me a DM on either discord or telegram (@zhayitong on both) i'm happy to loop our engineers in and help you integrate! cc @stepandel for visibility too

**Comment by Eithcowich** [comment]
Thanks @yitongzhang I sent you a DM on tg.
